transports/webrtc/src/tokio/certificate.rs

```diff
@@ -88,7 +88,7 @@ impl Certificate {
 }
 
 #[derive(thiserror::Error, Debug)]
-#[error("Failed to generate certificate")]
+#[error("Failed to generate certificate: {0}")]
```

Comment:
I don't think this should be necessary because `Kind` is going to be returned from `Error::source`.

Reply:
The reason I've made this change is because the error I got was `Failed to generate certificate` (no additional info)

Reply:
How did you print the error? Libraries like `anyhow` will iterate the source chain and print all errors. That seems like the cleaner approach to me although I do agree that the lack of this functionality in std is annoying.

Reply:
Do you feel strongly about including this change?

Reply:
not really. I'll remove